### DISCUSSION TITLE
Appendix for the organization's 'readme'

### DISCUSSION THREAD

**Discussion by Abhijay007** [keploy/keploy#75] [Ideas]
This discussion is meant to capture issues, questions, and requirements related to the implementation of the Appendix for the organization's 'readme' on the .github folder :)

To give you all a short overview that what this Appendix is for and how we can use .github folder, I write a short note here 👇 

Currently, keploy org repo does not have the special readme in the "overview section" to display the required info about keploy and different projects/repo we have under the org, Adding an Appendix and glossary section will help new visitors to learn more about the project in a short amount of time.

To know about the .github you can follow  [this](https://www.freecodecamp.org/news/how-to-use-the-dot-github-repository/) blog 

I request all the contributors to drop their suggestions and thoughts on it under this discussion :)

**Comment by nehagup** [comment]
Sounds good addition to me Abhijay! 

This will enhance the representability of Keploy Github org! Thanks for taking this up!👍

Would love to know what all can we add in.
- Must have
- Good to have

**Comment by Sarthak160** [comment]
Surely, It will be a good implementation and will really help new comers to know different repos and aspects of _Keploy_ _org_ in a quick and better way

Looking forward for it 🚀

**Comment by slayerjain** [comment]
@Abhijay007 Thanks for posting this.

I think this can help implement uniform policies for all our repos. Since we'll have SDKs in different languages and also examples.

LGTM! 😄

**Comment by unnati914** [comment]
maybe we can include contributing.md too

**Comment by Ayush7614** [comment]
@Abhijay007 @nehagup @Sarthak160 @unnati914 Hlo All Can i start working on this and @Abhijay007 it is very good idea

**Reply by nehagup** [reply]
Yes, definitely! Would love to see this in action.⭐

**Reply by Ayush7614** [reply]
> Yes, definitely! Would love to see this in action.⭐

so @nehagup please make a repository of name .github or you can give me access i can make

**Comment by nehagup** [comment]
Ref - https://github.com/keploy/.github/pull/1

**Comment by shub-garg** [comment]
Adding an Appendix and glossary section to the .github folder sounds like a great idea. It will provide new visitors with a quick overview of Keploy and various projects, making it easier to understand and get involved.

Looking forward to your ideas!

**Reply by Sonichigo** [reply]
We are working on creating a GitHub Wiki for the first time contributor/visitor to understand and get involved with keploy easily. 

I think it would be good to add their references to the org readme for better visibility.
ref:- [keploy wiki](https://github.com/keploy/keploy/wiki)

**Reply by shub-garg** [reply]
Yep!
